src/Attributes/src/Internal/Decorator.php

```diff
@@ -33,39 +35,39 @@ public function __construct(ReaderInterface $reader)
     /**
      * {@inheritDoc}
      */
-    public function getClassMetadata(\ReflectionClass $class, string $name = null): iterable
+    public function getClassMetadata(ReflectionClass $class, string $name = null): iterable
     {
         return $this->reader->getClassMetadata($class, $name);
     }
 
     /**
      * {@inheritDoc}
      */
-    public function getFunctionMetadata(\ReflectionFunctionAbstract $function, string $name = null): iterable
+    public function getFunctionMetadata(ReflectionFunctionAbstract $function, string $name = null): iterable
     {
         return $this->reader->getFunctionMetadata($function, $name);
     }
 
     /**
      * {@inheritDoc}
      */
-    public function getPropertyMetadata(\ReflectionProperty $property, string $name = null): iterable
+    public function getPropertyMetadata(ReflectionProperty $property, string $name = null): iterable
     {
         return $this->reader->getPropertyMetadata($property, $name);
     }
 
     /**
      * {@inheritDoc}
      */
-    public function getConstantMetadata(\ReflectionClassConstant $constant, string $name = null): iterable
+    public function getConstantMetadata(ReflectionClassConstant $constant, string $name = null): iterable
```

Comment:
Hello.
There are a lot of unnecessary changes. Can you turn off the rule to disable import of internal classes from global namespace?

Reply:
Yes, I can disable for short import classes:

via:

```php
$rectorConfig->disableImportShortClasses();
```

https://github.com/rectorphp/rector-src/blob/328c43713e7b7ec77675637582117619af012985/packages/Config/RectorConfig.php#L88

I may re-create PR for it when I have a chance

Reply:
I re-create PR for it at https://github.com/spiral/framework/pull/736